<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>766</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1043</width>
    <height>766</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1043</width>
    <height>766</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btn_basic">
        <property name="text">
         <string>图像基本操作</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_smooth">
        <property name="text">
         <string>阈值与平滑处理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>图像形态学处理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>图像梯度处理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>图像金字塔与轮廓检测</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>直方图与傅里叶变换</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>图像特征</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_8">
        <property name="text">
         <string>答题卡判卷</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_9">
        <property name="text">
         <string>全景图像拼接</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_10">
        <property name="text">
         <string>背景建模</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>5</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>751</width>
        <height>471</height>
       </rect>
      </property>
      <property name="title">
       <string>图像展示</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>430</y>
         <width>731</width>
         <height>21</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>原始图像</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>结果显示</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>919</x>
         <y>21</y>
         <width>16</width>
         <height>399</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>731</width>
         <height>401</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="img_origin">
          <property name="text">
           <string>原始图像显示</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="img_new">
          <property name="text">
           <string>处理结果显示</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>480</y>
        <width>761</width>
        <height>241</height>
       </rect>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>751</width>
          <height>231</height>
         </rect>
        </property>
        <property name="title">
         <string>操作选择</string>
        </property>
        <widget class="QPushButton" name="btn_merge">
         <property name="geometry">
          <rect>
           <x>600</x>
           <y>40</y>
           <width>119</width>
           <height>34</height>
          </rect>
         </property>
         <property name="text">
          <string>图像融合</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>109</y>
           <width>31</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>fx:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txt_fx">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>109</y>
           <width>51</width>
           <height>25</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_zoom">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>99</y>
           <width>119</width>
           <height>34</height>
          </rect>
         </property>
         <property name="text">
          <string>图像缩放</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>109</y>
           <width>41</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>亮度</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_20">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>160</y>
           <width>119</width>
           <height>34</height>
          </rect>
         </property>
         <property name="text">
          <string>降低</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_21">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>160</y>
           <width>119</width>
           <height>34</height>
          </rect>
         </property>
         <property name="text">
          <string>提高</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>170</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>对比度</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_graph">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>160</y>
           <width>119</width>
           <height>34</height>
          </rect>
         </property>
         <property name="text">
          <string>直方图均衡化</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_load">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>40</y>
           <width>119</width>
           <height>34</height>
          </rect>
         </property>
         <property name="text">
          <string>选择文件</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cbx_color">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>40</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>颜色通道提取</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Green</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Red</string>
          </property>
         </item>
        </widget>
        <widget class="QComboBox" name="cbx_edge">
         <property name="geometry">
          <rect>
           <x>400</x>
           <y>40</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>边界填充</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>复制法</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>反射法</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>反射法_101</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>外包装法</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>常量法</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>109</y>
           <width>31</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>fy:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txt_fy">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>109</y>
           <width>51</width>
           <height>25</height>
          </rect>
         </property>
        </widget>
        <widget class="QSlider" name="sld_bright">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>110</y>
           <width>281</width>
           <height>21</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="lbl_bri">
         <property name="geometry">
          <rect>
           <x>710</x>
           <y>100</y>
           <width>31</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>0%</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
